src/System.Diagnostics.DiagnosticSource/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
I've seen multiple discussions recently about these lock files getting checked in.  But since we're still on that plan, @ericstj, these are still supposed to be locked==true, right?